readme.md

```diff
@@ -987,9 +997,19 @@ test('rejects', async t => {
 });
 ```
 
+When testing an `async function` you must also wait for the assertion to complete:
+
+```js
+test('rejects', async t => {
+	await t.throws(async () => {
+		throw new TypeError('🦄');
+	});
+});
```

Comment:
Perhaps replace this example with the async function one you added above, and remove it there.